tasks/rss-mentor-dashboard.md

```diff
@@ -1,77 +1,83 @@
-# RSS Дашборд ментора
+# RSS Mentor Dashboard
 
 ### Deadline: Sunday, Feb 24, 2019 23:59
 
-## Задание
-Вы должны реализовать дашборд ментора школы The Rolling Scopes в виде странички на github pages.
+## Task
+You need to implement dashboard of The Rolling Scopes School mentor as a page on github pages.
 
-Дашборд должен содержать следующую информацию
-- список студентов, которых курирует ментор
-- список тасков студентов, которые должны делать студенты
-- таблица студент / задания должны быть подсвечена следующим образом:
-    - зеленый цвет - проверенные задания
-    - желтый цвет - задания сейчас в работе у студентов (status - in progress)
-    - красный цвет - задания, которые ментору надо проверять (status - checking)
-    - темно-красный цвет - задания, которые уже надо было сдать, но их не сдали (status - checked)
-    - серая подсветка / притухшие - задания, которые еще не надо делать (status - no status / todo)
+Dashboard should contain following information:
+- list of mentor's students
+- list of student's tasks, which students should work on
+- table student / tasks should be highlighted according to following rules:
+    - green color - reviewed tasks
+    - yellow color - tasks, which students are working on (status - in progress)
+    - red color - tasks, which are awaiting review from mentor (status - checking)
+    - dark red color - tasks, which should already be finished by students,
+    but they were not finished (status - checked)
+    - grey color - tasks, which should not be started yet (status - no status / todo)
```

Comment:
"yellow color is for tasks that students are working on now (status - in progress)"
"red color is for tasks that mentor should check (status - checking)"
"dark red color is for tasks that should already be done, but they are not (status - checked)"
"grey color/muted is  for tasks to be done (status - no status / todo)"

 sound better and less formal for me. . 
"That and which" topic -  https://www.myenglishteacher.eu/blog/which-vs-that/

Reply:
fixed